components/function-controller/README.md

```diff
@@ -66,3 +66,31 @@ To develop the component, use the formulae declared in the [generic](/common/mak
 | **WEBHOOK_DEFAULTING_MINREPLICAS**        | Value of the minReplicas which webhook should set if origin equals null                   | `1`                  |
 | **WEBHOOK_DEFAULTING_MAXREPLICAS**        | Value of the maxReplicas which webhook should set if origin equals null                   | `1`                  |
 | **WEBHOOK_DEFAULTING_RUNTIME**            | Value of the runtime which webhook should set if origin equals null                       | `nodejs14`           |
+
+## Troubleshooting
+
+### How to build and install `libgit2` binary, required by the Function Controller, on macOS
+
+1. Navigate to the Function Controller's root directory and verify the version of `git2go`:
+
+   ```bash
+   $ cat go.mod | grep git2go
+        github.com/libgit2/git2go/v31 v31.4.14
```

Comment:
Is the indentation intentional?

Reply:
I guess. It was like that in the [source issue](https://github.com/kyma-project/kyma/issues/13732).